packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -82,6 +88,7 @@ async function ensureNonceNonZero(
 ): Promise<void> {
   const nonce = await getNonce(connection, keyToAddress(identity.pubkey));
   const sendTx: SendTx = {
+    domain: "bns",
```

Comment:
Yeah, I was hoping this would be a domain for the `kind` (like isolating the class definitions), rather than a domain for the specific `object`.

So instead of `kind: "bcp/send"` and `kind: "bns/addUsername"`, we have

```ts
{
  domain: "bcp",
  kind: "send",
},
{
  domain: "bns",
  kind: "addUsername",
}
```

This allows us to avoid conflict in the use of kind in different packages, but doesn't add extra differentiation on the instances.

What do you think of this?